src/components/CarDetails/cardetailspage.jsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import CarDetails from './CarDetails';
+
+function CarDetailsPage() {
```

Comment:
- [ ] Kindly, refactor this function to ES6 arrow function.

Refactoring is easy:

```javascript
const CarDetailsPage = () => {
  // ...
}
```

So your code meets the latest standards. 👍